shm/.github/workflows/main.yml

```
@@ -0,0 +1,20 @@
+name: C++ with CMake CI
+
+on: [push, pull_request]
+
+jobs:
+  SHM:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@master
```

```suggestion
      uses: actions/checkout@SHM
```
Shouldn't this look like this since we don't have any code at master?

---

fixed!:)